.github/workflows/release.yml

```diff
@@ -0,0 +1,113 @@
+name: Release
+
+on:
+  - workflow_dispatch
+  # TODO: Temporary, should be removed before merged to main
+  - push
+
+permissions:
+  contents: read
+  # implicitly all other scopes not listed become none
+
+env:
+  BOTAN_REPO: randombit/botan
+  BOTAN_REF: 3.1.1
+
+jobs:
+  utility:
+    name: "Utility"
+    strategy:
+      fail-fast: false
+
+      matrix:
+        include:
+          - jobname: Documentation
+            target: pdf_docs
+            artifacts: |
+              build/docs/handbook/botan.pdf
+              build/docs/doxygen/
+            host_os: ubuntu-latest
+          - jobname: Coverage Report
+            target: coverage
+            artifacts: lcov-out/
+            host_os: ubuntu-latest
+
+    runs-on: ${{ matrix.host_os }}
+    steps:
+      - name: Fetch Audit Repository
+        uses: actions/checkout@v3
+        with:
+          path: ./source
+      - name: Fetch Botan Repository
+        uses: actions/checkout@v3
+        with:
+          path: ./botan
+          repository: ${{ env.BOTAN_REPO }}
+          fetch-depth: 0
+          ref: ${{ env.BOTAN_REF }}
+
+      - name: Setup Build Agent
+        uses: ./source/.github/actions/setup-build-agent
+        with:
+          target: ${{ matrix.target }}
+          cache-key: bsi-${{ matrix.host_os }}-${{ matrix.target }}
+
+      - name: Build and Test Botan
+        # TODO: Disabled tests is temporary fix for https://github.com/randombit/botan/pull/3632
+        run: >-
+          python3 ./source/.github/scripts/ci_build.py
+          --root-dir=${{ github.workspace }}/botan --build-dir=${{ github.workspace }}/build
+          --disabled-tests=tls_extensions_key_share_client_hello,xmss_keygen_reference
```

Comment:
`xmss_keygen_reference` could be fixed by enabling `shake`.

Reply:
`shake` is now enabled indirectly by enabling both kyber and sphincsplus.